### PR TITLE
add white list ddl filter

### PR DESCRIPTION
**Pull request by leoppro**
### What problem does this PR solve? 
add white list ddl filter, the user can specify the item in the white list by congig

### Check List 

Tests 

 - Unit test
 - Integration test

### CODE REVIEW

File: cmd/changefeed.toml [hunk 1]
```diff
@@ -1,6 +1,7 @@
-ignore-txn-commit-ts = []
 
 filter-case-sensitive = false
+ignore-txn-commit-ts = []
+ddl-white-list = [100, 101]
```
Comment (amyangfei) [inline cmd/changefeed.toml:4]: seems `100` and `101` are invalid DDL ActionType, recommend to use valid value. We should also update the document and put a URL here to tell user what value they can use

Comment (leoppro) [reply cmd/changefeed.toml]: updated, and I think we should hide `ddl-white-list` to user

**Pull request merged**